### PR TITLE
fix: update certification page padding-bottom

### LINKED ISSUE #41125: Certification page display issues - background and bottom padding
**Looking forward for reporting a security issue:**
Please report security issues by sending an email to `security@freecodecamp.org` instead of raising a GitHub issue.

**Describe the bug**
There is some display issues with the certification page, for example:

 - The background is set to height 100% which means it cuts off at strange places depending on the browser size.
 - There is no spacing at the end of the page so the last line of text just bumps into the bottom of the browser window

**To Reproduce**

1. [View a certificate page](https://www.freecodecamp.org/certification/rtpharry/responsive-web-design) (tested on Mac/Chrome)
2. Scale the browser window so it has a scrollbar
3. Scroll down and see that the white background cuts off (see screenshot 1 below)
4. Scroll to the end of the page to see that the page doesn't have any padding (see screenshot 2 below)

**Expected behavior**

**Screenshots**
Screenshot 1:
image

Screenshot 2:
image

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Mac
 - Browser: [e.g. chrome, safari] Chrome
 - Version: [e.g. 22] Version 88.0.4324.150 (Official Build) (x86_64)

**Additional context**

### PR DESCRIPTION
**Pull request by manziEric**
Checklist:

- [x ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `main` branch of freeCodeCamp.
- [ x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #41125

this pull request is to update the gray bottom of the page on different screen size with out making big change

### CODE REVIEW

**Review by raisedadead — DISMISSED** [review]
@ahmadabdolsaheb please confirm and merge if this OK.

**Review by moT01 — DISMISSED** [review]
I am seeing that this adds about a full page of padding at the bottom:
Screen Shot 2021-02-18 at 7 56 58 AM

I think maybe the spacer alone would be fine.

**Comment by ahmadabdolsaheb** [comment]
Removing the two padding bottoms would make it look good. 

Screen Shot 2021-02-18 at 5 38 00 PM

File: client/src/pages/certification.css [hunk 1]
```diff
@@ -186,6 +186,10 @@
     font-size: 15px;
     margin-top: 20px;
   }
+
+  .certificate-outer-wrapper {
+    padding-bottom: 55rem;
+  }
```
Comment (ahmadabdolsaheb) [inline client/src/pages/certification.css:192]:
```suggestion
```

File: client/src/pages/certification.css [hunk 2]
```diff
@@ -264,4 +268,5 @@
 .certificate-outer-wrapper {
   margin-top: calc(-1 * var(--header-height));
   background-color: #fff;
+  padding-bottom: 50rem;
```
Comment (ahmadabdolsaheb) [inline client/src/pages/certification.css:271]:
```suggestion
```

**Comment by manziEric** [comment]
@ahmadabdolsaheb the padding-bottom is removed from the css file

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**